jobs/rootfs-builder.jpl

```diff
@@ -106,18 +115,36 @@ node("debos && docker") {
     }
 
     j.dockerPullWithRetry(docker_image).
-        inside(" --privileged --device /dev/kvm ") {
+        inside(" --privileged --device /dev/kvm -v /dev:/dev") {
 
         stage("Build") {
             timeout(time: 8, unit: 'HOURS') {
 	        build(config, arch, pipeline_version, kci_core, rootfs_type)
             }
         }
 
+        stage("scp") {
+            sshagent(credentials : ['jenkins-chromeos-storage-private-key']) {
+                sh(script: """\
+scp \
+-o StrictHostKeyChecking=no \
+-P 22022 \
+-r \
+${pipeline_version}/_install_/* \
+chromeos@storage.chromeos.kernelci.org:\
+~/images/rootfs/chromeos/
+"""
+                  )
+            }
+        }
+    }
+/* Upload stage not working, as kci_rootfs http(s) upload fails for files >1GB */
+/*
         stage("Upload") {
             timeout(time: 30, unit: 'MINUTES') {
                 upload(config, pipeline_version, kci_core, rootfs_type, arch);
             }
         }
     }
+*/
```

Comment:
Maybe if scp works well enough we could start using it for all the rootfs builds and merge this in `main` branch for production? Are all the rootfs build jobs storing their artifacts in the same directory now or would this only work with Chrome OS builds?

Reply:
We can still merge this PR on `chromeos` branch first, I was just wondering if it would be trivial to "upstream" bits of it in `main`.

Reply:
Maybe no need to merge, i will just test it in state of separate branch in jenkins, and if it will work well, we can make separate, more clean and proper commit for main.